src/stories/index.js

```diff
@@ -10,28 +10,21 @@ import NavbarTopNotif from "../components/NavbarTopNotif"
 import NavbarTopProfile from "../components/NavbarTopProfile"
 import NavbarTopNotifBubble from "../components/NavbarTopNotifBubble"
 
-storiesOf("Page home", module).add("navbar", () => (
-  <div>
-    <Nav vertical>
-      <NavItem>
-        <NavLink href="#">Link</NavLink>
-      </NavItem>
-    </Nav>
-    <hr />
-  </div>
-))
+const Bubble0 = {
```

Comment:
is Bubble0 a component ? because that is the only reason to write it with a starting cap